berserker-http/README.md

```diff
@@ -0,0 +1,28 @@
+# Berserker HTTP
+
+Worker implementation which sends HTTP request on configured endpoint.
+
+Configuration can define two properties:
+1. `base-url` - can be concatenated with request property `url-sufix` to construct url.
+2. `headers` - contains header names with its values.
+
+Worker `accept` method expects following properties:
```

Comment:
Maybe to add expected response code (optional) so the worker can correctly know how to count successful and failed requests. Also, an optional request timeout.

Reply:
I'd leave this later, if needed.